openedx/features/course_experience/static/course_experience/js/Enrollment.js

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
When @andy-armstrong gets back, it would be great to have his opinion on ES2015 re-use and where something like this new code belongs.